### ISSUE TITLE
Add to documentation of how to add raw cells with ReST in JupyterLab

### ISSUE THREAD

**Issue by 1kastner** [spatialaudio/nbsphinx#625]
Hello there and thanks a lot for this great project! In #624, I quickly discussed how I used and plan to use ReST cells in my Jupyter Notebooks to create Sphinx-compatible links with the help of Sphinx roles. In the UI Jupyter Notebook, raw cells can be easily selected as it can be seen in the screenshot at https://nbsphinx.readthedocs.io/en/0.8.8/raw-cells.html. The support of Jupyter Notebook is fading (see [this announcement](https://github.com/jupyter/notebook#notice)) and more and more people shift to JupyterLab. Other UIs exist as well. Maybe it could be possible to extend the documentation at this point to show how to do the same (setting the Raw Cell type) in other UIs, such as JupyterLab? Or just provide a link to other sources where people can read it?

With my current JupyterLab version, it appears in the right sidebar.
![grafik](https://user-images.githubusercontent.com/5236165/148265230-87a640e3-6746-4509-bcbe-d98768f4e2a0.png)

If it is accepted, I could take a screenshot similar to https://github.com/spatialaudio/nbsphinx/blob/master/doc/images/raw_cells.png (does the depicted notebook reside in this repository?) and it could be added to the raw cell description - discussing both the Jupyter Notebook UI and JupyterLab. Would that be acceptable for this project?

**Comment by mgeier** [comment]
I'm painfully aware the the screenshot is becoming outdated.
However, I'm not willing to continuously update this screenshot (or even multiple of them).

If you want to update it (and/or add a different one for JupyterLab), I'd be happy to receive a PR from you.

Alternatively, the only thing I would be willing to change myself is to completely remove the screenshot because of the maintenance burden.

> does the depicted notebook reside in this repository?

Well, yes, it does (https://github.com/spatialaudio/nbsphinx/blob/master/doc/raw-cells.ipynb), but the screenshot depicts an old version.

That's yet another point to proof that screenshots will become outdated and have to be updated regularly ...

**Comment by 1kastner** [comment]
I can do an update this time for sure. I will also have a look whether there is a link (with intersphinx?) that could cross-reference to another documentation that could do the job here and the screenshots might not be necessary. I am not sure which one will create less work in the long run though.

**Comment by 1kastner** [comment]
Another idea that came a bit later - we could use selenium to automatically create the screenshot whenever the documentation builds. That would keep the screenshot up-to-date but it would increase the project dependencies. What do you think about that, @mgeier?

**Comment by 1kastner** [comment]
Now my first proposal for adjustment is out with #632. I added links to documentations I believe are rather stable. However, I did not find another source that could reliably do the job for this project. When it comes to raw cells, this repo is one of the first to pop up in the search engine by now.

**Comment by 1kastner** [comment]
I sticked with the idea of showing the notebook in the notebook because it is somehow funny. However, in order to see the Raw cells, I decided to scrol to the latter section. Is that fine with you?

![grafik](https://user-images.githubusercontent.com/5236165/148918982-bcb1313e-d6be-45fe-88a5-342b74301ef1.png)
![grafik](https://user-images.githubusercontent.com/5236165/148919004-c1f26c5b-95a0-46b1-bf63-47b7893b52db.png)
